cli/src/package_metadata.rs

```diff
@@ -0,0 +1,40 @@
+extern crate serde_derive;
+extern crate toml;
+
+use std::collections::HashMap;
+use serde_derive::Deserialize;
+use std::fs;
+
+#[derive(Deserialize)]
+pub struct PackageMetadata {
+  package: Package,
+  dependencies: HashMap<String, String>,
+}
+
+#[derive(Deserialize)]
+struct Package {
```

Comment:
why the split?

Reply:
the main struct holds both the package section and the dependencies section, if you have an idea for a better name shoot